### ISSUE TITLE
Google blocks Requests

### ISSUE THREAD

**Issue by telion2** [interactionlab/NLATool#138]
Recently I tested NLAtool and the GKG-Request got blocked by Google. 
Basically Google responded with a "Are you a Robot?" Screen, where a normal user has to mark every picture that fits a certain condition. Dont know why that happens now with an official Google API, but I will look into it.

**Comment by telion2** [comment]
https://stackoverflow.com/questions/51993001/google-blocks-requests-to-api

**Comment by telion2** [comment]
My guess is, that when reloading a text too often, Google get to many request which are the same and thus thinks that our app is spamming. We could do something about direckt reloading with a cookie, but if you load the same text with no initial cookie, I dont believe we have a solution without saving everything on the db. Or we use the db as a buffer and load the info from there if the text is loaded too often. 
If not solved, the appwill not show any Results for the Entities, and the user needs to prove to google that he is not a robot. Since I dont believe that the normal user is reloading the text that often, I guess its a low priority bug, that we maybe can warn the user about?

**Comment by telion2** [comment]
It seems that issue doesnt come up again.
